wiki/Do_you_really_want_to_ask_peppy/sr.md

```diff
@@ -0,0 +1,15 @@
+# Да ли стварно желите да питате peppy-ја?
+
+Здраво! peppy разуме твоју жељу да га лично контактираш, али тражи да прво схватиш да ово **није** место где желиш да га замолиш за помоћ или подршку. Он чува osu! подршку одвојену од личних порука и такође сматра да је доста тешко пратити PM-ове на форуму. Стога од вас тражи да урадите једну од следећих опција:
```

Comment:
```suggestion
Здраво! peppy разуме твоју жељу да га лично контактираш, али он жели да прво схватиш да ово **није** место где желиш да га замолиш за помоћ или подршку. Он чува osu! подршку одвојену од личних порука и такође сматра да је доста тешко пратити PM-ове на форуму. Стога од тебе тражи да урадиш једну од следећих опција:
```

Reply:
last sentence was changed because the first part looks to be non-formal but the words were written in a formal manner